app/src/main/java/com/android/unio/ui/event/overlay/PictureOverlay.kt

```diff
@@ -84,12 +187,11 @@ fun PictureOverlay(
                           .testTag(EventDetailsTestTags.EVENT_PICTURES_ARROW_LEFT),
                   colors =
                       IconButtonDefaults.iconButtonColors(
-                          contentColor = MaterialTheme.colorScheme.onPrimary,
-                          containerColor = MaterialTheme.colorScheme.primary)) {
+                          contentColor = MaterialTheme.colorScheme.onPrimary)) {
                     Icon(
                         Icons.AutoMirrored.Filled.ArrowBack,
                         context.getString(R.string.event_details_content_description_arrow_left),
-                        modifier = Modifier.size(iconSize))
+                        modifier = Modifier.size(arrowSize))
                   }
 
               IconButton(
```

Comment:
you could remove the arrow right if we are on the lastPage and remove the arrow left if we are on the first page, this should be done very easily with the pageState like `if pagerState.currentPageOffsetFraction == -0.5` for example